File: exercises/beecrowd/1070/1070.ts
// https://judge.beecrowd.com/pt/problems/view/1070
import * as fs from "fs";

function main(): void {
	const input = fs.readFileSync('/dev/stdin', 'utf8');
	const lines = input.split(/\r?\n/);

	const value = Number(lines.shift());

	let qtdResults = 0

	for (let i = value; qtdResults < 6; i++) {
		if (i%2 != 0) {
			qtdResults++
			console.log(i)
		}
	}
}

main();

File: exercises/beecrowd/1070/1070.go
// https://judge.beecrowd.com/pt/problems/view/1070
package main

import (
	"fmt"
)

func main() {
	var value int

	fmt.Scanf("%d", &value)

	qtdResults := 0

	for i := value; qtdResults < 6; i++ {
		if i%2 != 0 {
			qtdResults++
			fmt.Println(i)
		}
	}
}
